fiovb/host/main.c

```diff
@@ -115,6 +116,43 @@ static int write_persistent_value(const char *name, const char *value)
 	return -1;
 }
 
+static int delete_persistent_value(const char *name)
+{
+	char req1[MAX_BUFFER] = { '\0' };
+
+	struct fiovb_ctx ctx;
+	TEEC_Operation op;
+	TEEC_Result res;
+	uint32_t origin;
+
+	if (strlen(name) >= sizeof(req1)) {
+		fprintf(stderr, "Name value too large (should be up to %d)\n",
+				sizeof(req1));
+		return -1;
+	}
+
+	prepare_tee_session(&ctx);
```

Comment:
Will fix in a follow up commit.

Reply:
Looking a bit more, I think it should be fine to stay the way we have now because that function will exit in case of failures in TEEC_InitializeContext and TEEC_OpenSession (errx).

Since the logic is small enough and the code won't really do much in case it can't open a context/session, it should be fine to exit in case of an error there.